qemu/tests/cfg/cpu_topology_test.cfg

```diff
@@ -13,3 +13,12 @@
             required_qemu = [6.2.0,)
             # Check PPTT table mapping by looking at core_id
             check_core_id_cmd = 'cat /sys/devices/system/cpu/cpu%s/topology/core_id'
+        s390x:
+            no RHEL.6 RHEL.7
+            no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8
+            required_qemu = [8.2.0-1,)
+            RHEL8:
+                check_siblings_cmd_rhel8 = 'core_siblings_count=$(($(cat /sys/devices/system/cpu/cpu0/topology/core_siblings_list | cut -d'-' -f2) - $(cat /sys/devices/system/cpu/cpu0/topology/core_siblings_list | cut -d'-' -f1) + 1)); echo "$core_siblings_count"'
+            RHEL.9:
+                check_siblings_cmd = 'cat /proc/cpuinfo |grep siblings |uniq |cut -d ":" -f 2'
```

Comment:
Can use the same variable name in RHEL8 and RHEL9.

Reply:
updated